acceptance-tests/src/test/java/hudson/plugins/gradle/MockGeServer.java

```diff
@@ -0,0 +1,118 @@
+package hudson.plugins.gradle;
```

Comment:
just for my piece of mind, can you tripe check with Luke that it's OK to expose publicly this kind of information.
It's not really secret, and can probably be found by folks inspecting the traffic between the GE instance and build agents, but just to be sure

Reply:
Yeah I thought about the same but this is not secret as these are the endpoints the build agents call and are quite easy to capture. But still good to double check.